client/components/ProfileHeader.js

```diff
@@ -0,0 +1,120 @@
+// Module imports
+import React from 'react'
+// import PropTypes from 'prop-types'
+
+
+
+
+// Component imports
+import { connect } from '../store'
+import { formatAsEliteDateLong } from '../helpers/formatTime'
+import {
+  selectUser,
+  selectUserDisplayRat,
+  selectUserAvatar,
+} from '../store/selectors'
+
+
+
+
+// Component Constants
+
+
+
+
+
+@connect
+class ClassName extends React.Component {
+  /***************************************************************************\
+    Class Properties
+  \***************************************************************************/
+
+  state = {}
+
+
+
+
+
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
+  _renderUserGroups = () => this.props.user.relationships.groups.data.map((item) => <li className={`badge ${item.id}`} key={item.id}>{item.id}</li>)
```

Comment:
Please split this up a little. At least where it becomes JSX so it's not all forced onto one line.
Kind of like:
```suggestion
  _renderUserGroups = () => this.props.user.relationships.groups.data.map((item) => (
    <li className={`badge ${item.id}`} key={item.id}>
      {item.id}
    </li>
  ))
```

Though that might not be enough. If the user has no groups, data will most likely be null. You should probably run a null check before mapping.